City-of-Pori/pori-kada
# PORI-372 Fixed styling issue with visitpori English menu

To test:

1. drush cc all
2. Go to local.visitpori.fi/en and make sure the menu looks like it has been styled the same way as the menu on the Finnish side. Just don't forget that there is less menu items and they are probably not color coded yet (you need to add class to menu item such as visitpori-theme-color-orange to make the item change color.
Remember to test both mobile and desktop.
